edx/edx-ora2
# EDUCATOR-4827: Toggle rubric visibility for learners

**TL;DR -** Add an ORA setting to allow users to see a collapsable grading rubric while they work on their response

**What changed?**

- Added a field to ORA, `show_rubric_during_response`, that can be set to true or false in the ORA settings in studio
- Added functionality to oa_rubric.html to be rendered in "read-only" mode, which looks the same except that all input elements are hidden. 
- When `show_rubric_during_response` is set to true, render the rubric in read-only mode in a collapsable container at the top of the response section

**Testing Instructions**

- Create an ORA. In the settings page, scroll down to the bottom and set `Show Rubric During Response` to be true.
- Publish the ORA, and then open it as a learner
- The `What will this assignment be graded on?` section should appear in the Response section, and clicking it should display the rubric with no inputs (radio buttons, text inputs, etc)

![Screen Shot 2021-02-19 at 11 46 52 AM](https://user-images.githubusercontent.com/1639231/108535927-d4ca4f80-72a9-11eb-9873-755497543a3c.png)

![Screen Shot 2021-02-19 at 11 49 19 AM](https://user-images.githubusercontent.com/1639231/108535897-cd0aab00-72a9-11eb-9c9e-e2b74100e45d.png)

![Screen Shot 2021-02-19 at 11 49 27 AM](https://user-images.githubusercontent.com/1639231/108535868-c5e39d00-72a9-11eb-87f2-931b5790240f.png)


**Developer Checklist**
- [ ] UI/UX review

JIRA: [EDUCATOR-4827](https://openedx.atlassian.net/browse/EDUCATOR-4827)

FIY: @edx/masters-devs-gta
